libzecale/circuits/pairing/group_variable_gadgets.hpp

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2015-2021 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
```

Comment:
Let's skip a line after this header for consistency

Reply:
Inserted line.